Makefile

```diff
@@ -380,6 +380,7 @@ SOUND_BIN_DIR  := $(BUILD_DIR)/sound
 TEXTURE_DIR    := textures
 ACTOR_DIR      := actors
 LEVEL_DIRS     := $(patsubst levels/%,%,$(dir $(wildcard levels/*/header.h)))
+VNL_ACTRS_DIRS := $(patsubst actors/vanilla_actors/%,%,$(dir $(wildcard actors/vanilla_actors/*/header.h)))
```

Comment:
is there any reason this could not just be added to ACTOR_DIR?

Reply:
I prefer "vanilla objects system" being explicitly separate.